Documentation/Concepts/Cache/Index.rst

```diff
@@ -0,0 +1,58 @@
+:navigation-title: Cache
+..  include:: /Includes.rst.txt
+
+..  _cache:
+
+=====
+Cache
+=====
+
+.. _what-is-caching:
+
+What is Caching in TYPO3?
+-------------------------
```

Comment:
Headlines of level two should have `====` as underline. Please see https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Menus/HeadlinesAndSection.html For a complete list of header styles.